python_module/CMakeLists.txt

```diff
@@ -29,20 +29,32 @@ if(CREATE_PYTHON_MODULE)
     target_link_libraries(${libname} PRIVATE ${MKL_DEF_LIBRARY})
   endif()
 
+  set(PYTHON_INSTALL_SITE_DIR ${CMAKE_INSTALL_PREFIX}/lib/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages)
```

Comment:
I suggest you use `${CMAKE_INSTALL_LIBDIR}` instead of `lib`, sometimes it's lib64. This can simplify `relDir` to `python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages` and you can just set it directly.

Reply:
relDir should be `$ORIGIN/../../../`. I let cmake compute it instead of hardcoding it.